app/lib/netem_commands.js

```diff
@@ -45,6 +45,9 @@ exports.build = function (netemPort, operation) {
       commands.push({cmd: "sudo tc filter add dev "+iface+" parent "+
                           "ffff: protocol ip u32 match u32 0 0 flowid 1:1 "+
                           "action mirred egress redirect dev "+ifb_iface});
+      commands.push({cmd: "sudo tc filter add dev "+iface+" parent "+
+                          "ffff: protocol ipv6 u32 match u32 0 0 flowid 1:1 "+
```

Comment:
is `u32` still right here? I figured that was magic because of the length of an IPv4 address.

Reply:
seems like its probably right, and I just don't know what that u32 means.

Reply:
From `tc-u32(8)`:

```
SELECTORS
       Basically the only real selector is u32 .  All others merely provide a higher level syntax and are internally translated into u32 .
```

I think this just matches everything, same as the ip filter